pipelines/deployer/deployer.yaml

```diff
@@ -833,6 +833,44 @@ jobs:
       GOOGLE_OAUTH_CLIENT_SECRET: ((google-oauth-client-secret))
       PLATFORM_RESOURCE_TYPE: ((platform-resource-type))
       PLATFORM_VERSION: ((platform-version))
+  - task: apply-cluster-config-resources
+    image: task-toolbox
+    timeout: 10m
+    config:
+      platform: linux
+      inputs:
+      - name: config
+      params:
+        ACCOUNT_ROLE_ARN: ((account-role-arn))
+        AWS_DEFAULT_REGION: eu-west-2
+        AWS_REGION: eu-west-2
+        CLUSTER_NAME: ((cluster-name))
+        DEFAULT_NAMESPACE: gsp-system
+      run:
+        path: /bin/bash
+        args:
+        - -c
+        - |
+          set -euo pipefail
+          if [ -d "config/cluster-resources" ]
```

Comment:
Should this be one directory for all clusters in a given config repo, or different directories for each cluster?

Reply:
I don't feel strongly either way. I opted for the simplest option first. If we ever need the "per-cluster" variant we can always change it later. At this point that's only the case for the sandbox account, and given the tech-ops strategy...

Reply:
I suppose so. And given the only other clusters there are on-demand with potential for custom branches of the config repo, that's fine.